ports/libdatachannel/vcpkg.json

```
@@ -1,11 +1,12 @@
 {
   "name": "libdatachannel",
-  "version-semver": "0.17.10",
+  "version-semver": "0.17.10-1",
```

```suggestion
  "version-semver": "0.17.10",
  "port-version": "1",
```
Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).